flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -975,6 +981,32 @@ private int handleArgException(Exception e) {
 	}
 
 	/**
+	 * Displays an optional exception message for incorrect program parametrization.
+	 *
+	 * @param e The exception to display.
+	 * @return The return code for the process.
+	 */
+	private int handleParametrizationException(ProgramParametrizationException e) {
+		String message = e.getMessage();
+		if (message != null) {
+			System.err.println(message);
+		}
```

Comment:
This block could be removed if we got rid of the zero-args constructor in `ProgramParametrizationException`.